File: src/ELibrary.Backend/ShopApi/Domain/Dtos/Order/ManagerUpdateOrderRequest.cs
using LibraryShopEntities.Domain.Entities.Shop;

namespace ShopApi.Domain.Dtos.Order
{
    public class ManagerUpdateOrderRequest
    {
        public int Id { get; set; }
        public string DeliveryAddress { get; set; }
        public DateTime DeliveryTime { get; set; }
        public OrderStatus OrderStatus { get; set; }
    }
}

File: src/ELibrary.Frontend/src/app/modules/shared/domain/dtos/shop/order/managerUpdateOrderRequest.ts
import { Order, OrderStatus } from "../../../.."

export interface ManagerUpdateOrderRequest {
    id: number,
    deliveryAddress: string,
    deliveryTime: Date,
    orderStatus: OrderStatus
}

export function mapOrderToManagerUpdateOrderRequest(order: Order): ManagerUpdateOrderRequest {
    return {
        id: order.id,
        deliveryAddress: order.deliveryAddress,
        deliveryTime: order.deliveryTime,
        orderStatus: order.orderStatus
    }
}